pipeline/outputs/opentelemetry.md

```diff
@@ -20,7 +20,7 @@ Important Note: At the moment only HTTP endpoints are supported.
 | traces_uri                  | Specify an optional HTTP URI for the target web server listening for traces, e.g: /v1/traces | /         |
 | header               | Add a HTTP header key/value pair. Multiple headers can be set. |           |
 | log_response_payload | Log the response payload within the Fluent Bit log           | false     |
-| add_label            | This allows you to add custom labels to all metrics exposed through the OpenTelemetry exporter. You may have multiple of these fields |           |
+| add_label            | This allows you to add custom labels to all metrics and logs (resource attributes) exposed through the OpenTelemetry exporter. You may have multiple of these fields |           |
```

Comment:
How do we specify multiple labels? Comma-separated string, can we escape characters?

I'm not sure if this is using a common function in the code, in which case we should align the docs with wherever else that uses it, e.g. Loki or Prometheus output?

Reply:
For multiple labels one add multiple "add_label" in the config, this is the same as the current behavior (and implied in the example of the [doc](https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry). All the escaping logic should also be the same as before which I haven't touch. Its just that before the PR only metrics use the config data, this PR makes it be used by logs.

I don't think a common function is used here either (though this might be able to be refactored to make it common to other plugins). The current implementation convert the labels at create phase to a another format ready to be appended, then it uses in `process_metrics`. My PR modifies `process_logs` to make use of the converted label vales and embed it in the open-telemetry log structure.

Reply:
OK, can you indicate in the docs this option should be repeated for each label you want to add? Essentially let's just clarify if we can.